build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -537,9 +537,18 @@ message Command {
     string value = 2;
   }
 
-  // The arguments to the command. The first argument must be the path to the
-  // executable, which must be either a relative path, in which case it is
-  // evaluated with respect to the input root, or an absolute path.
+  // The arguments to the command.
+  //
+  // The first argument specifies the command to run, which may be either an
+  // absolute path, a path relative to the working directory, or an unqualified
+  // path (without path separators) which will be resolved using the operating
+  // system's equivalent of the PATH environment variable. Path separators
```

Comment:
Maybe specify if this takes into account `PATH` env variables specified in the request and what happens if no such variable is specified.

Reply:
+1

Reply:
I have added the following:

> If the `environment_variables` list contains an entry for the PATH environment variable, it SHOULD be respected. If not, the resolution process is implementation-defined.

I'm not too happy with introducing implementation-defined behaviour, but at least it matches POSIX:

https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/basedefs/V1_chap08.html#tag_08_03

> If PATH is unset or is set to null, the path search is implementation-defined.